FishBun/build.gradle

```diff
@@ -39,13 +39,13 @@ apply from: '../gradle/release.gradle'
 
 dependencies {
     compileOnly "androidx.appcompat:appcompat:$rootProject.androidx_version"
-    compile "androidx.constraintlayout:constraintlayout:$rootProject.constraint_version"
-    compile "com.google.android.material:material:$rootProject.material_version"
-    compile "androidx.recyclerview:recyclerview:$rootProject.recyclerview_version"
+    implementation "androidx.constraintlayout:constraintlayout:$rootProject.constraint_version"
```

Comment:
Why did you change this?
There was an issue when i received a dependency using an implementation, need to check that.
So it's better not to go into this change.

Reply:
Because compile has been deprecated so that's why i have changed it. will revert it wait

Reply:
Also can you let me know what issue you were facing with implementation maybe i can help you with that too

Reply:
When uploading to the maven repository, it was a problem with dependencies. I will check this.